src/systems/filecoin_vm/interpreter/vm_interpreter.go

```diff
@@ -225,6 +226,29 @@ func (vmi *VMInterpreter_I) ApplyMessage(
 	return
 }
 
+// Ensures a messages's receiving actor exists in the state tree.
+// If it doesn't, and the message is a simple value send to a pubkey-style address,
+// creates the receiver as an account actor in the returned state.
+func _ensureReceiver(tree st.StateTree, message msg.UnsignedMessage) (st.StateTree, bool) {
+	_, found := tree.GetActor(message.To())
+
+	if found {
+		return tree, true
+	}
+	if !message.To().IsKeyType() {
+		// Don't implicitly create an account actor for an address without an associated key.
+		return tree, false
+	}
+	if message.Method() != actor.MethodSend {
+		// Don't implicitly create account actor if message was expecting to invoke a method.
+		return tree, false
+	}
+	// TODO Create a new account actor via the InitActor, which will receive a new ID address
```

Comment:
nit: would be better to use `util.TODO()` or `panic` here, to make absolutely clear that this TODO is mandatory

Reply:
The implementation is coming in the direct follow-up, so I beg off this time.